code/desktop/src/desktop/DesktopLauncher.java

```diff
@@ -1,20 +1,21 @@
 package desktop;
 
-import com.badlogic.gdx.Files;
-import com.badlogic.gdx.backends.lwjgl.DungeonApplication;
-import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
+import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application;
+import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration;
 import controller.LibgdxSetup;
 import controller.MainController;
 import tools.Constants;
 
 public final class DesktopLauncher {
     public static void run(MainController mc) {
-        LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
-        config.width = Constants.WINDOW_WIDTH;
-        config.height = Constants.WINDOW_HEIGHT;
-        config.foregroundFPS = Constants.FRAME_RATE;
-        config.title = Constants.WINDOW_TITLE;
-        config.addIcon(Constants.LOGO_PATH, Files.FileType.Internal);
-        new DungeonApplication(new LibgdxSetup(mc), config, 0);
+        Lwjgl3ApplicationConfiguration config = new Lwjgl3ApplicationConfiguration();
+        config.setWindowSizeLimits(Constants.WINDOW_WIDTH, Constants.WINDOW_HEIGHT, 1000, 1000);
+        // maxWidth: 1000 and maxHeight: 1000 affects the fullscreen behavior
```

Comment:
> Aber eine Frage wäre dann noch offen, wie wir mit LWJGL3 die Fenstergröße einstellen können.

Könnten wir dazu hier nicht einfach 
```suggestion
        config.setWindowedMode(Constants.WINDOW_WIDTH, Constants.WINDOW_HEIGHT);
```

setzen um die Fenstergröße in der Lwjgl3config zu setzen?

Reply:
Geht leider nicht, weil Lwjgl3ApplicationConfiguration die Methode setWindowSizeLimits haben möchte und man bei dieser Methode 4 Parameter angeben muss. Aber dieses 1000 betrifft nur die maximale Größe des Fensters, wenn man es vergrößert.

Reply:
Aber, wenn man es im Fullscreen spielen möchte, könnte man die größtmögliche Auflösung setzen (9999), wenn man auf maximieren klickt, dann wird das Fenster nicht größer als die Bildschirmauflösung...

Kann ich noch ändern.

Reply:
Also wenn ich bei dem branch in der config:

`config.setWindowSizeLimits(Constants.WINDOW_WIDTH, Constants.WINDOW_HEIGHT, 1000, 1000);`
auskommentiere und dafür:
`config.setWindowedMode(Constants.WINDOW_WIDTH, Constants.WINDOW_HEIGHT);`
setze startet der dungeon bei mir ganz normal und ich könnte ihn auch maximieren um die volle Bildschirmfläche zu benutzen. Aber ich glaube das ist nicht das was Du eigentlich möchtest, oder?

Reply:
Sound funktioniert im fix_mac branch auch. Voll gut.